### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem
- *#9*
- *#20*
- *No change_url script*

## Solution
- *Add change_url script*
- *Add `extra_php-fpm.conf` with `php_admin_value[memory_limit] = 256M`*
- *Install missing icones*

## PR Status
- [x] Code finished.
- [x] Tested with Package_check.
- [ ] Fix or enhancement tested.
- [ ] Upgrade from last version tested.
- [x] Can be reviewed and tested.

## Package_check results
---

[Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/matomo_ynh%20PR24%20(ericgaspar)/)

### CODE REVIEW

File: scripts/upgrade [hunk 1]
```diff
@@ -80,8 +81,24 @@ if [ "$upgrade_type" == "UPGRADE_APP" ]
...
-	ynh_setup_source --dest_dir="$final_path"
+	if [[ ! -f "$final_path/config/config.ini.php" ]]; then
+		ynh_setup_source --dest_dir="$final_path"
+	else
+		# Create a temporary directory
+		tmpdir="$(mktemp -d)"
+
+		# Backup the config file in the temp dir
+		cp -a "$final_path/config/config.ini.php" "$tmpdir/config.ini.php"
+
```
Comment (yalh76) [inline scripts/upgrade:92]: why not removing `final_path` before installing new source ?

File: scripts/upgrade [hunk 2]
```diff
@@ -118,16 +134,34 @@ ynh_script_progression --message="Installating composer and dependencies..."
...
+	popd
+fi
+
+#=================================================
+# INSTALL MISSING ICONS
+#=================================================
+
+if [ ! "$(ls -A "$final_path/plugins/Morpheus/icons")" ]
+then
+    git clone https://github.com/matomo-org/matomo-icons.git "$final_path/plugins/Morpheus/icons" --quiet
```
Comment (yalh76) [inline scripts/upgrade:149]: why not using something like `ynh_setup_source --dest_dir="$final_path/plugins/Morpheus/icons" --source_id="icons"` adn having an icons.src file ?

File: scripts/install [hunk 3]
```diff
@@ -96,43 +96,26 @@ phpversion=$(ynh_app_setting_get --app=$app --key=phpversion)
 #=================================================
 # INSTALL COMPOSER AND DEPENDENCIES
 #=================================================
-ynh_script_progression --message="Installing composer and dependencies..."
+ynh_script_progression --message="Installing Composer and dependencies..."
 
 ynh_install_composer --phpversion="$phpversion" --workdir="$final_path"
 
+# Install missing icons
+git clone https://github.com/matomo-org/matomo-icons.git "$final_path/plugins/Morpheus/icons" --quiet
```
Comment (yalh76) [inline scripts/install:104]: why not using something like ynh_setup_source --dest_dir="$final_path/plugins/Morpheus/icons" --source_id="icons" adn having an icons.src file ?

**Review by yalh76 — CHANGES_REQUESTED** [review]

File: scripts/upgrade (hunk 2 above)
Comment (ericgaspar) [reply scripts/upgrade]: Thanks @yalh76 for reviewing this PR. I added the download from `ynh_setup_source`

**Review by yalh76 — APPROVED** [review]

**Pull request merged**